app/src/main/java/de/westnordost/streetcomplete/screens/main/map/StyleableOverlayManager.kt

```diff
@@ -40,6 +43,8 @@ class StyleableOverlayManager(
 
     private val viewLifecycleScope: CoroutineScope = CoroutineScope(SupervisorJob())
 
+    private var styledElementsUpdateJob: Job? = null
```

Comment:
how about just naming it updateJob? After all, we are already in this class, it's clear that it is for the styled elements. (Same with the other class.)

Reply:
done